proto/v3/message_contents/public_key.proto

```diff
@@ -34,6 +34,17 @@ message VmacAccountLinkedKey {
     }
 }
 
+// Keys linked to an account (e.g. signed by a wallet)
+// The purposes of the keys are encoded in the signatures
+message VmacAccountLinkedKeys {
```

Comment:
This creates a new type for the keys, but I don't see where this will include it in the contact bundle.

Reply:
It might make sense to copy `VmacInstallationPublicKeyBundleV1` to a `V2` struct and just replace the `identity_key` with something like `installation_keys`. The V1 struct can just be deleted later

Reply:
Given we are offering no guarantees of compatibility right now, I'd also be OK with just replacing V1 with the new type.

Reply:
Keeping old versions around isn't all that helpful if we don't plan to have a SDK available that can read them

Reply:
I thought of that too, but with that approach as soon as we call `gen_protos.sh` we have to immediately refactor to use the new type in all locations in the codebase. With the separate type, we can replace V1 with V2 in pieces in the code until there's no V1 left, and then delete the V1 proto, so we don't have any baggage.

Reply:
We aren't going to have to update in one pass regardless? Seems like things will be pretty broken with half the contact handling code on V1 and half on V2. 

If we introduce a v2 and then delete v1, there would be an extra step of renaming v2 to v1 at the end. Or we just launch with only a v2 proto and a reserved v1 slot lives there forever.